osu.Game/Overlays/Settings/Sections/Graphics/LayoutSettings.cs

```diff
@@ -145,9 +146,7 @@ protected override void LoadComplete()
             {
                 updateResolutionDropdown();
 
-                const string not_fullscreen_note = "Running without fullscreen mode will increase your input latency!";
-
-                windowModeDropdown.WarningText = mode.NewValue != WindowMode.Fullscreen ? not_fullscreen_note : string.Empty;
+                windowModeDropdown.WarningText = mode.NewValue != WindowMode.Fullscreen ? GraphicsSettingsStrings.NotFullscreenNote : string.Empty;
```

Comment:
Can use `default` rather than `string.Empty` in such cases (saves on an implicit `string -> LocalisableString` conversion).

```suggestion
                windowModeDropdown.WarningText = mode.NewValue != WindowMode.Fullscreen ? GraphicsSettingsStrings.NotFullscreenNote : default;
```